src/server/game/Entities/Player/Player.h

```diff
@@ -2386,7 +2389,9 @@ class TC_GAME_API Player : public Unit, public GridObject<Player>
         uint32 m_questRewardTalentCount;
 
         // Social
-        PlayerSocial* m_social;
```

Comment:
This is a bad change, don't bloat `Player` class

Reply:
Maybe you could've used the old PlayerSocial, just improve/refactor/optimize it, seems better to me than throwing new methods/properties onto this huge thing that calls itself Player

Reply:
Marking this PR as Feedback-FixOutdatedMissingWIP until this change is reverted